### DISCUSSION TITLE
如何修改复选框/任务列表的样式？

### DISCUSSION THREAD

**Discussion by Celetherin** [CaiJimmy/hugo-theme-stack#757] [Q&A] [answered]
您好！
现在的checkbox好像使用的是浏览器默认样式，如下
![image](https://user-images.githubusercontent.com/113590856/211091653-3c805aa8-ddf0-4f45-9bbb-74db9d6afd1c.png)
我想知道它能够通过css样式进行自定义么？
我尝试在normaliaze.scss以及custom.scss中修改过，不过都没有生效。
```html
  input[type="checkbox"]:disabled{
    background-color: transparent;
    color: var(--card-text-color-main);
    border-color: var(--card-text-color-main);
  }
```
我没什么代码知识，这个也是求助AI写出来的，希望能够得到解答，感谢！

**Comment by CaiJimmy** [comment, marked as answer]
> 我想知道它能够通过css样式进行自定义么？

主题没有自定义 `checkbox` 的样式，所以没有样式冲突的问题。

关于如何自定义它的样式，我也不是很清楚，抱歉。

**Reply by Celetherin** [reply]
没关系，也很感谢您的回复啦！
